sample-files/jsonDataProviderUserClass.java

```diff
@@ -0,0 +1,90 @@
+public class USER {
```

Comment:
USER is the example class name we already have in place.
This is a class purely in here for website documentation so we don't waste extra space with package, imports, etc.